templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -218,6 +218,36 @@ export class {{ service.name }}Client {
     };
 {%- endif %}
 
+{%- if (service.bundleConfigs.length > 0) %}
+    // Some methods on this API support automatically batching
+    // requests; denote this.
+{%- if (service.longRunning.length === 0) %}
```

Comment:
To reduce copy-paste, can we instead make it the part of the long running condition?

https://stackoverflow.com/questions/42677401/nunjucks-if-with-multiple-and-or-or-condition shows how to do `or` in Nunjucks.

Reply:
done! :)